app/modules/import-export-customization/assets/js/import/pages/import-complete.js

```diff
@@ -0,0 +1,156 @@
+import { Box, Typography, Card, CardContent, Button, Link, Stack } from '@elementor/ui';
+import { useNavigate } from '@reach/router';
+import { __ } from '@wordpress/i18n';
+import { useEffect } from 'react';
+import { BaseLayout, TopBar, PageHeader, Footer } from '../../shared/components';
+import { useImportContext } from '../context/import-context';
+
+const Illustration = () => (
+	<Box
+		sx={ {
+			height: 140,
+			alignSelf: 'center',
+			borderRadius: 2,
+			display: 'flex',
+			alignItems: 'center',
+			justifyContent: 'center',
+		} }
+	>
+		<img
+			src={ elementorAppConfig.assets_url + 'images/kit-is-live.svg' }
+			alt="Kit is live illustration"
+		/>
+	</Box>
+);
+
+const ExternalLinkIcon = () => (
+	<Box
+		component="i"
+		sx={ { fontFamily: 'eicons' } }
+		className="eps-icon eicon-editor-external-link"
+	/>
+);
+
+export default function ImportComplete() {
+	const { isCompleted } = useImportContext();
+	const navigate = useNavigate();
+
+	const headerContent = (
+		<PageHeader title={ __( 'Import', 'elementor' ) } />
+	);
+
+	const footerContent = (
+		<Stack direction="row" spacing={ 1 }>
+			<Button
+				variant="outlined"
+				color="secondary"
+				startIcon={ <ExternalLinkIcon /> }
+				size="small"
+				data-testid="see-it-live-button"
+			>
+				{ __( 'See it Live', 'elementor' ) }
+			</Button>
+			<Button
+				variant="contained"
+				color="primary"
+				size="small"
+				sx={ { px: 4 } }
+				data-testid="close-button"
+			>
+				{ __( 'Close', 'elementor' ) }
+			</Button>
+		</Stack>
+	);
+
+	useEffect( () => {
+		if ( ! isCompleted ) {
+			navigate( '/import-customization', { replace: true } );
+		}
+	}, [ isCompleted, navigate ] );
+
+	return (
+		<BaseLayout
+			topBar={ <TopBar>{ headerContent }</TopBar> }
+			footer={ <Footer>{ footerContent }</Footer> }
+		>
+			<Box
+				display="flex"
+				flexDirection="column"
+				alignItems="center"
+				mx="auto"
+				pt={ 10 }
+				gap={ 4 }
+			>
+				<Illustration />
+				<Stack spacing={ 1 } alignItems="center">
+					<Typography
+						variant="h4"
+						color="text.primary"
+					>
+						{ __( 'Your website templates is now live on your site!', 'elementor' ) }
+					</Typography>
+					<Typography
+						variant="body1"
+						color="text.secondary"
+					>
+						{ __( 'You\'ve imported and applied the following to your site:', 'elementor' ) }
+					</Typography>
+				</Stack>
+				<Card sx={ { borderRadius: 2, boxShadow: 0, border: '1px solid', borderColor: 'elevation.outlined' } }>
```

Comment:
Do we need a `<Card>`? I found it doesn't look good by default in Dark Mode. Maybe you can just use a `Box`.

Reply:
Does it have a black background? Darker than the rest of the screen.